app/models/concerns/user_concern.rb

```diff
@@ -144,11 +152,11 @@ def self.available_permission_contexts
     end
 
     def self.anyone_using_acls?
-      active.not_system.where(permission_context: 'acls').exists?
+      active.not_system.using_acls.exists?
     end
 
     def self.all_using_acls?
-      ! active.not_system.where(permission_context: [nil, 'role_based']).exists?
+      ! active.not_system.using_role_based.exists?
```

Comment:
Hmm, so, OP has actually used the space after the bang for negation since it's much easier to see the negation.  Perhaps we should discuss this and standardize.